### PR TITLE
feat: Update uniswap provider to use new websocket api

### PR DESCRIPTION
**Pull request by zarazan**
## Description

- The old uniswap provider was using a polling mechanism to retrieve prices from a product called "The Graph". This has been replaced with a websocket API similar to the osmosis and kujira APIs. It is a strait up copy paste from the osmosis provider replacing "osmosis" with "uniswap".
- Updated the config to remove unnecessary uniswap config values

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -24,7 +24,6 @@ linters:
     - gocritic
     - gofmt
     - goimports
-    - gosec
```
Comment (adamewozniak) [inline .golangci.yml:27]: Why were these bits removed?

File: ojo-provider-config/currency-pairs.toml [hunk 2]
```diff
@@ -164,13 +158,6 @@ providers = [
 ]
 quote = "OSMO"
 
-[[currency_pairs]]
```
Comment (adamewozniak) [inline ojo-provider-config/currency-pairs.toml:167]: We might want to put these configs in a separate PR just to avoid conflicts when we merge this into the umee branch

But, those conflicts would be pretty easy so up to you 🤷

**Review by adamewozniak — APPROVED** [review]
a couple comments but lgtm!

File: .golangci.yml (hunk 1 above)
Comment (zarazan) [reply .golangci.yml:27]: It complains about any URLs in the provider files that don't start with https://. I was going to add another exclusion but didn't think it was helpful.

File: ojo-provider-config/currency-pairs.toml (hunk 2 above)
Comment (zarazan) [reply ojo-provider-config/currency-pairs.toml:167]: Not my first conflicts on an umme backport :). No problem, easier to have one review and not leave the repo in a broken state.

File: .golangci.yml (hunk 1 above)
Comment (zarazan) [reply .golangci.yml]: I added it back in and made another exclusion rule.
